### PR TITLE
Documentation to include inclusive learning materials.

### PR DESCRIPTION
**Pull request by dstarr**
## Purpose
Adds a navigable set of learning materials to the repository. This is the start of what can be grown into even more resources.

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/dstarr/commercial-marketplace-samples.git
cd commercial-marketplace-samples
git checkout starr
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* page layout is acceptable
* Spelling and punctuation

### CODE REVIEW

File: docs/learning-ama.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- [Recorded webinars](#webinars)
+
+## Key Documentation
+
+  - [Plan an Azure Application offer for the commercial marketplace](https://docs.microsoft.com/en-us/azure/marketplace/plan-azure-application-offer)
+  - [Publishing guide for Azure managed applications](https://docs.microsoft.com/en-us/azure/marketplace/marketplace-managed-apps)
+  
+## Instructional Videos
+
+The first two videos are meant to be consumed as a pair. Both are described in [this blog post](https://techcommunity.microsoft.com/t5/fasttrack-for-azure/simple-azure-managed-application-creating-testing-and-publishing/ba-p/2071773). The first focuses on developing an Azure Managed Applicaiton and the second looks at pubishing it as a transactable offer in the Azure Marketplace.
```
Comment (scseely) [inline docs/learning-ama.md]: Please run this through a spell checker. "Pubishing" should be "publishing". I expect that similar spelling errors exist elsewhere. https://marketplace.visualstudio.com/items?itemName=ban.spellright

**Comment by dstarr** [comment]
Closing pull request due to spelling errors.

File: docs/learning-ama.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+1. [Creating and testing a simple Azure Managed Application in the developer’s Azure subscription](https://www.youtube.com/watch?v=eCGFmUirLGU)
+2. [Publishing Azure Managed App in Partner Center and testing in end-customer’s Azure subscription](https://www.youtube.com/watch?v=3ODHzEjomro)
+
+[Mastering Azure Managed Applications](https://www.youtube.com/playlist?list=PLmsFUfdnGr3xpz1Q1ExBxj2Dk_dWqzGB6) is an 18-part instructional video series. It is a recording of a full day hands-on workshop held May 11, 2021. The workshop videos have been organized into logical modules to make each one short and digestible, with a specific focus. This is a 100-400 level workshop, so most of your questions about Azure Managed Applications should be answered in this video series. [The labs conducted in the training](http://aka.ms/AMAWorkshopLabs) are available for you to use at anytime.
```
Comment (scseely) [inline docs/learning-ama.md:21]: "The lab is conducted"

File: docs/learning-ama.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Webinars
+
+> Technical webinars for the Azure Marketplace are conducted every 2 weeks. You may [sign up for a live session here](https://microsoftcloudpartner.eventbuilder.com/MarketplaceDeveloperOfficeHours).
```
Comment (scseely) [inline docs/learning-ama.md:25]: Use the https://aka.ms/MarketplaceDeveloperOfficeHours alias instead. It's possible that we'll change the URL to something else;)

File: docs/learning-general.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# General Transactable Offer Learning
+
+## Webinars
+
+> Technical webinars for the Azure Marketplace are conducted every 2 weeks. You may [sign up for a live session here](https://microsoftcloudpartner.eventbuilder.com/MarketplaceDeveloperOfficeHours).
```
Comment (scseely) [inline docs/learning-general.md:5]: Use the https://aka.ms/MarketplaceDeveloperOfficeHours alias instead. It's possible that we'll change the URL to something else;)

File: docs/learning-saas.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+## Key Documentation
+
+- [How to plan a SaaS offer for the commercial marketplace](https://docs.microsoft.com/en-us/azure/marketplace/plan-saas-offer)
+- [Build the landing page for your transactable SaaS offer in the commercial marketplace](https://docs.microsoft.com/en-us/azure/marketplace/azure-ad-transactable-saas-landing-page)
+- [SaaS fulfillment APIs version 2 in the commercial marketplace](https://docs.microsoft.com/en-us/azure/marketplace/partner-center-portal/pc-saas-fulfillment-api-v2)
+- [Metered billing for SaaS using the commercial marketplace metering service](https://docs.microsoft.com/en-us/azure/marketplace/partner-center-portal/saas-metered-billing)
+
+## Webinars
+
+> Technical webinars for the Azure Marketplace are conducted every 2 weeks. You may [sign up for a live session here](https://microsoftcloudpartner.eventbuilder.com/MarketplaceDeveloperOfficeHours):
```
Comment (scseely) [inline docs/learning-saas.md:17]: Use the https://aka.ms/MarketplaceDeveloperOfficeHours alias instead. It's possible that we'll change the URL to something else;)

File: docs/learning-vm.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+## Instructional Videos
+
+The following two videos show two different ways of publishing a Virtual Machine on the Azure Marketplace. Both are described in [this blog post](https://techcommunity.microsoft.com/t5/fasttrack-for-azure/simple-azure-marketplace-virtual-machine-offer-creating-vm/ba-p/2058093). The recommended technique is to use the Azure Shared Image Gallery.
+
+1. [Create Azure VM image for Azure Marketplace Virtual Machine Offer using Packer with VHD destination](https://www.youtube.com/watch?v=AcsXUuYCvQw)
+2. [Creating Azure VM image for Azure Marketplace Virtual Machine Offer using Packer with Azure Shared Image Gallery](https://www.youtube.com/watch?v=FVPiecI5r3s)
+
+## Webinars
+
+> Technical webinars for the Azure Marketplace are conducted every 2 weeks. You may [sign up for a live session here](https://microsoftcloudpartner.eventbuilder.com/MarketplaceDeveloperOfficeHours).
```
Comment (scseely) [inline docs/learning-vm.md:25]: Use the https://aka.ms/MarketplaceDeveloperOfficeHours alias instead. It's possible that we'll change the URL to something else;)

**Pull request closed without merging**